### ISSUE TITLE
Forefoot pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11183]
**Preferred term label:**
Forefoot pain

**Synonyms**
https://www.ncbi.nlm.nih.gov/medgen/C0025587
Metatarsalgia; 

**Definition (free text, please give PubMed ID)**

Burning, aching, or sharp pain about the metatarsal heads in the forefoot

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001376
- label: Forefoot pain
- parents: [Foot pain (HP:0025238)](https://hpo.jax.org/app/browse/term/HP:0025238)
- pmid: PMID:28109624
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized about the metatarsal heads in the front part of the foot.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11183
- synonyms: Metatarsalgia(NONE)
